scripts/exsrw_prepstart.sh

```diff
@@ -118,7 +118,7 @@ else
       echo "${PDY}${cyc}: cycle smoke/dust from ${checkfile} "
     fi
 
-    ${USHsrw}/add_smoke.py
+    ${USHsrw}/smoke_dust/add_smoke.py
```

Comment:
@BenKoziol-NOAA, I understand why you try to make a new sub-directory for smoke_dust here. However, it might not a good idea for the NCO EE2 compliance perspective. The 'ush' directory should contain the utility scripts called from the ex-scripts. For each application, the unnecessary scripts, not used by the application, should be removed from the app for the EE2 compliance. In case that the smoke/dust app is delivered to NCO, other AQM-related scripts should be removed. In this case, we should remove this smoke_dust dir and move the scripts to ush again. So, please do not create this sub-dir 'smoke_dust' under ush. It would be good to put all utility scripts in ush together.

Reply:
@chan-hoo I did wonder if the directory structure would go against NCO standards and was going to ask before putting this up for review. I'll plan to remove the directory and prefix the scripts with `smoke_dust_*` instead.

Reply:
@chan-hoo I made the prefix change and reverted the directory move. The `black` code formatting touched a lot of lines in the scripts. I think the change is a good one in terms of readability and maintainability. However, I would not view these changes (renaming, formatting) as absolutely necessary. Furthermore, they can be applied at anytime, so if it you don't think it makes sense to merge, that's totally understandable.

Reply:
It should be fine with me if the scripts work well :)

Reply:
Okay! Let me give it one more once-over and a test on Orion, and I'll open it up for review.

Reply:
@chan-hoo My tests are clean on Orion. I'm going to open this for review.